cylc/rose/utilities.py

```diff
@@ -451,9 +482,9 @@ def get_cli_opts_node(opts=None, srcdir=None):
         {'value': 'BAZ', 'state': '', 'comments': []}
     """
     # Unpack info we want from opts:
-    opt_conf_keys = []
-    defines = []
-    rose_template_vars = []
+    opt_conf_keys: list = []
+    defines: list = []
+    rose_template_vars: list = []
```

Comment:
Can it be more specific? `List[str]`?

Reply:
Don't know the types, added the `list` placeholder purely to allow the function to by type checked in the first place.